sdks/go/examples/wordcap/wordcap.go

```diff
@@ -31,7 +30,7 @@ import (
 )
 
 var (
-	input = flag.String("input", os.ExpandEnv("$GOPATH/src/github.com/apache/beam/sdks/go/data/haiku/old_pond.txt"), "Files to read.")
+	input = flag.String("input", "gs://apache-beam-samples/shakespeare/kinglear.txt", "File(s) to read.")
```

Comment:
Does this still work on the direct runner with this change? Without all the GCP specific flag stuff?

Reply:
Let me try to clarify the code here and purpose of this example. It already runs on Dataflow. This flag is deliberately local and super-small (old_pond). textio.Immediate embeds it into a transforms, so that is a straight-line IO-free pipeline that just logs the output of that local file. Its purpose was to debug the Go SDK at first and later the Flnk portable runner.

Multiout is in a similar bucket. It may rather make sense to move them out of the way of the normal examples?

Reply:
Can we remove those examples instead of moving them out of normal examples? Do they still serve a purpose?

Reply:
Sort of. They might be useful for new runners (ULR, Spark,..), but integration tests can serve the same purpose.

Reply:
To be clear: I'm fine with these examples being deleted.

Reply:
Agreed. I will start by deleting this one.